src/npc_attack.h

```diff
@@ -77,8 +77,9 @@ class npc_attack_melee : public npc_attack
 class npc_attack_gun : public npc_attack
 {
         const gun_mode gunmode;
+        item &gun;
```

Comment:
wait, why? you get that from gunmode!

Reply:
Tried to make the gunmode member non-const but it still complained about const-ness whenever I tried to get the weapon with *gunmode. It seems to always choose the const version of the function over the non-const.